web/frontend/src/ui/AppHeader.tsx

```diff
@@ -0,0 +1,23 @@
+import * as React from 'react';
+import { connect } from 'react-redux';
+import { NavLink } from 'react-router-dom';
+import {AppBar, Box, Button, Toolbar, Typography} from "@material-ui/core";
+
+class HeaderImpl extends React.Component {
+    render() {
+        return (
+            <Box sx={{ flexGrow: 1, height: '6%'}}>
```

Comment:
Why is height in `%`?

Reply:
Because with this height value, Header looks normal.